src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -127,9 +127,10 @@ private Uri CreateAuthorizationUri(bool addVerifier = false, bool addState = fal
             if (addVerifier)
             {
                 _codeVerifier = PlatformPlugin.CryptographyHelper.GenerateCodeVerifier();
-                string codeVerifierHash = PlatformPlugin.CryptographyHelper.CreateSha256Hash(_codeVerifier);
+                string codeVerifierHash = MsalHelpers.EncodeToBase64Url(
```

Comment:
this is because of [https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/289](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/289)
With the next fix, this code will look exactly how it looked before

---
In reply to: [109345485](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/282#discussion_r109345485) [](ancestors = 109345485)